lang/cs/Org.Apache.REEF.Tang/Interface/IConfigurationBuilder.cs

```diff
@@ -28,6 +28,15 @@ public interface IConfigurationBuilder
         IClassHierarchy GetClassHierarchy();
         IConfiguration Build();
         void Bind(string iface, string impl);
+
+        /// <summary>
+        /// Bind an implementation to an interface with alias language
+        /// </summary>
+        /// <param name="iface"></param>
+        /// <param name="impl"></param>
+        /// <param name="language"></param>
```

Comment:
Make sense. As it involves multiple file changes, I would do it in a separate Jira. REEF-962 is created.